dspace/src/main/docker/test/rest_web.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Could we add a comment to the top of this file that describes its purpose?  I'm just worried we'll forget why we are overriding this REST web.xml with a new one